test/client/index_test.rb

```diff
@@ -431,8 +437,8 @@
           @index.docs.index(document)
         end
 
-        assert_equal 400, exception.status
-        assert_match /\[output\]/, exception.message
+        assert_equal( 400, exception.status)
```

Comment:
Yeah this kind of thing is why I prefer parenthesis more often than not. Great commit message 😂 

Sorry to style snipe you but do you mind deleting the space after the parens here?

Reply:
ah crap yeah you got it thanks!